src/jvm/test/resources/expected_logs/varhandle_representation.txt

```diff
@@ -0,0 +1,228 @@
+|       boolFlag.compareAndSet(false,true): true at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:82)                                   |                                                                                                                                                                          |
+|       boolFlag.set(false) at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:83)                                                        |                                                                                                                                                                          |
+|       doubleNumber.READ: 1.0 at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:85)                                                     |                                                                                                                                                                          |
+|       doubleNumber.compareAndSet(1.0,2.0): true at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:85)                                  |                                                                                                                                                                          |
+|       doubleNumber.set(3.0) at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:86)                                                      |                                                                                                                                                                          |
+|       floatNumber.READ: 1.0 at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:88)                                                      |                                                                                                                                                                          |
+|       floatNumber.compareAndSet(1.0,2.0): true at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:88)                                   |                                                                                                                                                                          |
+|       floatNumber.set(3.0) at VarHandleRepresentationTest.instanceFieldHandles(VarHandleRepresentationTests.kt:89)                                                       |                                                                                                                                                                          |
```

Comment:
Is it possible to have a set of small tests instead? Let's also make a hierarchy of the test outputs in the `resources` folder.

Reply:
Done